src/core/solvers/kkt.cpp

```diff
@@ -13,26 +13,27 @@ namespace crocoddyl {
 
 SolverKKT::SolverKKT(boost::shared_ptr<ShootingProblem> problem)
     : SolverAbstract(problem),
-      regfactor_(10.),
-      regmin_(1e-9),
-      regmax_(1e9),
+      reg_incfactor_(10.),
+      reg_decfactor_(10.),
+      reg_min_(1e-16),
+      reg_max_(1e12),
```

Comment:
follows https://github.com/loco-3d/crocoddyl/pull/893#discussion_r542122288

Reply:
I already answer this above.
Let's discuss there. I will keep it open only for my own use